app/controllers/posts_controller.rb

```diff
@@ -1,6 +1,6 @@
 class PostsController < ApplicationController
   def index
-    @user = User.find(params[:user_id])
+    @user = User.includes(:comments, :posts).find(params[:user_id])
   end
 
```

Comment:
- [ ] It would be better if we fix the Syntax error in our part of the code, In the postsController, in line 13 and 32, there is a syntax error caused as a result of a user creating a post. Kindly find screenshot below
![Screenshot from 2022-03-31 23-14-11](https://user-images.githubusercontent.com/46388113/161164786-3e4cd462-9046-43f8-b5d1-fccee390ac8a.png)

Reply:
Hello @addod19,

The creation of a post works fine with me, you just have to have a user already in the database and it should work.
I get no syntax errors while creating posts. You may get logic errors if you have no users but that's about it, no syntax errors on my end.